src/components/HelloWorld.vue

```diff
@@ -1,55 +1,139 @@
 <template>
   <div class="hello">
-    <h1>{{ msg }}</h1>
+    <h1>{{ "This is My Bad Ass Page. I Can Do All Things." }}</h1>
```

Comment:
the handlebars aren't needed for this, that message could be bound to a data object.  I would replace "This...Things" with {{msg}} and pass that in with the component reference.